Modul_6_Dzhumelya/js/index.js

```diff
@@ -0,0 +1,90 @@
+// Вправа №1 Копія масива через метод concat()
+
+const arr = ['one', 10, 'name', 5, 'Nastya', 9];
+
+function arrayCopy () {
+    return arr.concat();
+}
+const newCopyArr = arrayCopy();
+console.log(newCopyArr);
+
+// Вправа №2 
+
+const arrTwo = ['The', 'quick', 'brown', 'fox', 'jumps', 'over', 'the', 'lazy', 'dog'];
+
+function arrayJoin (arr) {
```

Comment:
Допишіть функцію таким чином, щоб вона приймала розділювач як вхідний параметр
```suggestion
function arrayJoin (arr, delimiter) {
```
`const arrNewJoin = arrayJoin(arrTwo, ' '); `